src/gui/windows/pause.zig

```diff
@@ -15,11 +15,21 @@ pub var window = GuiWindow {
 
 const padding: f32 = 8;
 
+fn exitMenuCallbackFunction(useless: usize) void {
+	_ = useless; 
```

Comment:
No need to put this into a new line, better directly discard it with `fn exitMenuCallbackFunction(_: usize) void {`